src/applications/hca/tests/utils/helpers.unit.spec.js

```diff
@@ -259,6 +260,34 @@ describe('hca helpers', () => {
     });
   });
 
+  describe('isOfCollegeAge', () => {
+    it('returns `false` if birthdate is greater than 23 years from testdate', () => {
```

Comment:
Why is 23 the cutoff? I went to college after my military stent, when I was 25 😭

Reply:
Business rules for the form. This is an ongoing discussion with the stakeholder, as you could enter college before 18 as well.